gc/base/MemorySubSpaceSemiSpace.cpp

```diff
@@ -1114,5 +1114,21 @@ MM_MemorySubSpaceSemiSpace::mergeLargeObjectAllocateStats(MM_EnvironmentBase *en
 	_largeObjectAllocateStats->mergeCurrent(_memorySubSpaceSurvivor->getMemoryPool()->getLargeObjectAllocateStats());
 }
 
+uintptr_t
+MM_MemorySubSpaceSemiSpace::releaseFreeMemoryPages(MM_EnvironmentBase* env)
+{
+	return releaseFreeMemoryPages(env, 0);
```

Comment:
this should inline what the other one does (release memory for both allocate and survivor), obviously without any memory type check